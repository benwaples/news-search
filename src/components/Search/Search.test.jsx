import React from 'react';
import { 
  cleanup, 
  fireEvent, 
  render, 
  screen, 
  waitFor 
} from '@testing-library/react';
import Search from './Search';

describe('Search component', () => {
  afterEach(() => cleanup());

  it('should render a search input', () => {
    const { asFragment } = render(<Search 
      searchTerm="portland"
      onChange={() => {}}
    />);

    expect(asFragment()).toMatchSnapshot();
    
  });
});
