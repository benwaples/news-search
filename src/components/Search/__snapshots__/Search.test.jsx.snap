// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search component should render a search input 1`] = `
<DocumentFragment>
  <div
    id="Search"
  >
    <label
      for="search"
    >
      Search
    </label>
    <input
      id="search"
      name="search"
      type="search"
      value="portland"
    />
  </div>
</DocumentFragment>
`;
